evidence/wirednkod/0001-2024-05-evidence.md

```diff
@@ -0,0 +1,42 @@
+# Evidence-0001: Retain at Rank 1
+
+|                 |                                                                         |
+| --------------- | ----------------------------------------------------------------------- |
+| **Report Date** | 11 May 2024                                                             |
+| **Submitted by**| Nikos Kontakis                                                          |
+
+## Member details
+
+- Matrix username: @wirednkod:matrix.org
+- Polkadot address: 15DCZocYEM2ThYCAj22QE4QENRvUNVrDtoLBVbCm5x4EQncr
+- Current rank: 1
+- Date of initial induction: Seeding
+- Date of last report: N/A
+- Area(s) of Expertise/Interest: Light Clients/Polkadot-SDK/UX
+
+
+## Reporting period
+
+- Start date: YYYY/MM/DD
```

Comment:
Please indicate a start date for this reporting period, knowing that the report should cover a minimum of **6 months**.